client/core/core.go

```diff
@@ -1150,8 +1144,9 @@ func (c *Core) verifyRegistrationFee(wallet *xcWallet, dc *dexConnection, coinID
 		return confs >= uint32(reqConfs), nil
 	}
 
-	c.wait(assetID, trigger, func(err error) {
-		log.Debugf("Registration fee txn %s now has %d confirmations.", coinIDString(assetID, coinID), reqConfs)
+	regFeeAssetID, _ := dex.BipSymbolID(regFeeAssetSymbol)
```

Comment:
Since we're currently only using dcr for fee payments, thought to simplify the `verifyRegistrationFee` method signature by removing the `assetID` param that we is pretty much already a global variable.

If a change is later made to use some other asset for fees, the assetID would likely be in `core.*` or if made dex-configurable, accessible from `dc.cfg.*`, we still should not need to pass an assetID parameter.